signer/Cargo.toml

```diff
@@ -34,16 +34,20 @@ reqwest.workspace = true
 serde.workspace = true
 serde_json.workspace = true
 sha2.workspace = true
+sqlx.workspace = true
 thiserror.workspace = true
-time = "0.3"
+time.workspace = true
 tonic.workspace = true
-tokio = { workspace = true, features = ["signal"] }
+tokio = { workspace = true, features = ["signal", "macros", "rt-multi-thread"] }
 tokio-stream.workspace = true
 tracing.workspace = true
 tracing-attributes.workspace = true
 tracing-subscriber.workspace = true
 wsts.workspace = true
 
+# Only for testing
+fake = { version = "2.9.2", features = ["derive", "time"], optional = true }
```

Comment:
Because I want to be able to use it in the integration test, which technically is an external crate so it can't access any functionality hidden behind `#[cfg(test)]` - that's why I used the `#[cfg(testing)]` flag instead to gate this functionality.

If you have other preferences, I'm happy to reorganize this.

Reply:
I just stuck out, but right, that makes sense. Does that mean that the fake crate gets included in the release binary due to the testing being part of the default feature?

Reply:
I imagine that we'll build the release binary with `--no-default-features`, in which case the fake crate won't get included.

Reply:
Oh, I didn't know about that, but it makes sense that it exists.